### ISSUE TITLE
TypeError: Cannot read properties of null (reading 'removeAllListeners')

### ISSUE THREAD

**Issue by johnsonnc** [ebaauw/homebridge-soma#14]
Hi, 
New install seeing this error:
```
[13/12/2021, 19:44:55] [SOMA] warning: c69695add0e9: request 12: disconnected unexpectedly
[13/12/2021, 19:44:55] TypeError: Cannot read properties of null (reading 'removeAllListeners')
    at SomaPeripheral.destroy (/usr/lib/node_modules/homebridge-soma/lib/BleClient.js:579:22)
    at SomaClient.<anonymous> (/usr/lib/node_modules/homebridge-soma/lib/SomaPlatform.js:156:22)
```

```
Homebridge v1.3.8 (HAP v0.9.7) (Homebridge 1AD8) is running on port 51667.
[SOMA] hardware: Raspberry Pi 4B 1.4 (8GB)
[SOMA] os: Raspbian GNU/Linux 11 (bullseye)
[SOMA] bluetooth [E4:5F:01:49:C7:7D] enabled
```

Using the latest: 1.10.0-0 although I had the issue with 1.9.x

Thanks Nick!

**Comment by ebaauw** [comment]
Thanks.  That's easily fixed by a check for `null`.  Still feel like Noble is playing tricks on me...

**Comment by ebaauw** [comment]
Should be fixed in beta v1.0.10-1.

**Comment by johnsonnc** [comment]
Awesome - loaded it up - seems to have cleared up.
